ui/app/mainui/activitycenter/views/ActivityNotificationNewDevice.qml

```diff
@@ -0,0 +1,149 @@
+        property string assetBgColor: ""
+        property string ctaText: ""
+        property var actionSourceComponent: undefined
+
+        readonly property string desktopAssetName: "desktop"
+    }
+
+    bodyComponent: RowLayout {
+        spacing: 8
+
+        StatusSmartIdenticon {
+            Layout.preferredWidth: 40
+            Layout.preferredHeight: 40
+            Layout.alignment: Qt.AlignTop
+            Layout.leftMargin: Style.current.padding
+            Layout.topMargin: 2
+
+            asset {
+                width: 24
+                height: width
+                name: d.assetName
+                color: d.assetColor
+                bgWidth: 40
+                bgHeight: 40
+                bgColor: d.assetBgColor
+            }
+        }
+
+        ColumnLayout {
+            spacing: 2
+            Layout.alignment: Qt.AlignTop
+            Layout.fillWidth: true
+
+            StatusMessageHeader {
+                Layout.fillWidth: true
+                displayNameLabel.text: d.title
+                timestamp: root.notification.timestamp
+            }
+
+            RowLayout {
+                spacing: Style.current.padding
+
+                StatusBaseText {
+                    Layout.fillWidth: true
+                    text: d.info
+                    font.italic: true
+                    wrapMode: Text.WordWrap
+                    color: Theme.palette.baseColor1
+                }
+
+                Loader { sourceComponent: d.actionSourceComponent }
+
+            }
+        }
+    }
+
+    ctaComponent: undefined
+
+    states: [
+        State {
+            when: root.type === ActivityNotificationNewDevice.InstallationType.Received
+            PropertyChanges {
+                target: d
+                title: qsTr("New device detected")
+                info: qsTr("New device with %1 profile has been detected.").arg(accountName)
+                ctaText: qsTr("More details")
+                assetColor: Theme.palette.primaryColor1
+                assetBgColor: Theme.palette.primaryColor3
+                assetName: d.desktopAssetName
+                actionSourceComponent: ctaFlatBtnComponent
+            }
+        },
+        State {
+            when: root.type === ActivityNotificationNewDevice.InstallationType.Created
+            PropertyChanges {
+                target: d
+                title: qsTr("Sync your profile")
+                info: qsTr("Check your other device for a pairing request.")
+                ctaText: qsTr("More details")
+                assetColor: Theme.palette.primaryColor1
+                assetBgColor: Theme.palette.primaryColor3
+                assetName: d.desktopAssetName
+                actionSourceComponent: ctaFlatBtnComponent
+            }
+        }
+    ]
+
+    Component {
+        id: ctaFlatBtnComponent
+
+        StatusFlatButton {
+            size: StatusBaseButton.Size.Small
+            text: d.ctaText
```

Comment:
Same here, the `ctaText` is identical, so put the whole component directly under `d` with the same text